pages/components/grid/grid.css

```diff
@@ -0,0 +1,55 @@
+.grid {
+    display: grid;
+    gap:0.75rem;
+    width: 45rem;
+    height: 7.5rem;
+}
+
+.grid_two_items {
+    grid-template-areas: 
+    "Item_one Item_two"
+    "Item_three Item_four";
+}
+
```

Comment:
you can improve on naming classes. The class name is `two_items` but it contains four items inside. This sounds inconsistent.